src/main/kotlin/no/nav/syfo/model/ReceivedDialogmelding.kt

```diff
@@ -14,4 +20,53 @@ data class ReceivedDialogmelding(
     val mottattDato: LocalDateTime,
     val legehpr: String?,
     val fellesformat: String,
-)
+) {
+    companion object {
+        fun create(
```

Comment:
Flyttet logikken fra `DialogmeldingProcessor` og til domeneobjektet slik at man kunne bruke det litt rundt omkring uten å dra inn processor-klassen.

Reply:
Kan evt også legge på `private constructor` hvis vi ønsker at `create()` alltid skal brukes for å lage ny objekter.